PROCESSES.md

```diff
@@ -0,0 +1,147 @@
+# Processes, release procedures etc
+
+This repository includes multiple projects at multiple versions,
+some of which depend on each other. That makes things tricky, and
+leads to rather more complicated procedures than a simple "single
+package" repository. Please read this guide carefully to understand
+how releases happen, and why they happen that way.
+
+## The API catalog
+
+Only the source within the `apis` directory is published, and not
+all of that. (There are test projects and tools alongside the
```

Comment:
"not all of that" isn't too clear. Maybe something like "...is published, although not every project under that directory."